pkg/cloud/services/networking/network.go

```diff
@@ -139,6 +138,7 @@ func (s *Service) ReconcileNetwork(clusterName string, openStackCluster *infrav1
 		}
 	}
 
+	record.Eventf(openStackCluster, "SuccessfulCreateNetwork", "Created network %s with id %s", networkName, network.ID)
```

Comment:
If adding tags fails, network is created but no event recorded.
Current event recording is better.

Reply:
I agree. If it's really important that we have another event after the tagging we could consider producing two events. One after network creation and one after tagging. But I'm not sure what the use case is for this and thus what we can achieve with it.